tests/test_utils.py

```diff
@@ -1,32 +1,60 @@
 # -*- coding: utf-8 -*-
-import unittest
+import pytest
 import rdflib
 from skosprovider.exceptions import ProviderUnavailableException
-from skosprovider_getty.utils import uri_to_graph, SubClassCollector
+from skosprovider_getty.utils import (
+    uri_to_graph,
+    SubClassCollector,
+    GVP,
+    ISO
+)
 
-
-class UtilsTests(unittest.TestCase):
+class TestUtils(object):
 
     def test_uri_to_graph(self):
         uri = 'http://vocab.getty.edu/aat/300007466.rdf'
         res = uri_to_graph(uri)
-        self.assertIsInstance(res, rdflib.graph.Graph)
+        assert isinstance(res, rdflib.graph.Graph)
 
     def test_uri_to_graph2(self):
         uri = 'http://vocab.getty.edu/aat/300007466'
-        self.assertRaises(TypeError, uri_to_graph, uri)
+        with pytest.raises(TypeError):
+            res = uri_to_graph(uri)
 
     def test_uri_to_graph_not_found(self):
         uri = 'http://vocab.getty.edu/aat55/300zzz7466.rdf'
         res = uri_to_graph(uri)
-        self.assertFalse(res)
+        assert not res
 
     def test_uri_to_graph_error(self):
         uri = 'http://teeezssst.teeteest.test/aat55/300zzz7466.rdf'
-        self.assertRaises(ProviderUnavailableException, uri_to_graph, uri)
+        with pytest.raises(ProviderUnavailableException):
+            res = uri_to_graph(uri)
+
+    def test_get_subclasses(self):
+        from rdflib.namespace import SKOS
```

Comment:
Het was hiervoor ook al, maar, is er een reden om de imports in de methods te doen?

Reply:
Ik geloof dat de redenering is, indien een import faalt, dan faalt enkel deze test ipv de ganse testsuite.